### DISCUSSION TITLE
Default install.packages command in R-universe

### DISCUSSION THREAD

**Discussion by louisaslett** [r-universe-org/help#525] [General]
Big fan of R-universe here!

Just a tiny query really: each package page lists the install.packages which enables pulling from R-universe with a CRAN repo fallback for getting dependencies, I assume.  So, for example, one of my packages it [looks like this here](https://louisaslett.r-universe.dev/mlmc):

```
install.packages("mlmc", repos = c("https://louisaslett.r-universe.dev", "https://cloud.r-project.org"))
```

I was just wondering why the cloud repo is forced?  One of the nice things about R is the strong mirror ecosystem, and I personally use a UK academic mirror which is hosted on the same national network and even faster therefore than the cloud one for me.  Is there a reason the default command is not:

```
install.packages("mlmc", repos = c("https://louisaslett.r-universe.dev", options("repos")[[1]]))
```

which would respect the user's personal mirror choices?

Thanks for the awesome work on R-universe!

**Comment by jeroen** [comment]
This is mostly just the minimal suggested command for novice users. Pro users like you can adapt it to their needs. But what you use won't work for everyone because some installations have nothing preset for `options("repos")[[1]]` or something that is not cran.

I personally don't use random mirrors because R does not verify integrity of downloads. The cloud mirror is safe and fast everywhere.

**Comment by louisaslett** [comment]
Thanks @jeroen , some interesting info I wasn't aware of there.  The default makes total sense in light of that! 👍
